claat/nodes/youtube_test.go

```diff
@@ -0,0 +1,71 @@
+package nodes
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestNewYouTubeNode(t *testing.T) {
+	tests := []struct {
+		name string
+		in   string
+		out  *YouTubeNode
+	}{
+		{
+			name: "Empty",
+			out: &YouTubeNode{
+				node: node{typ: NodeYouTube},
+			},
+		},
+		{
+			name: "NonEmpty",
+			in:   "Mlk888FiI8A",
+			out: &YouTubeNode{
+				node:    node{typ: NodeYouTube},
+				VideoID: "Mlk888FiI8A",
+			},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			out := NewYouTubeNode(tc.in)
```

Comment:
Consider renaming `in` to `videoID`

Reply:
Discussion on similar comment https://github.com/googlecodelabs/tools/pull/596/files/591b1ac68607383d635229cd063ed8561fc4b93d